docker-compose.yml

```diff
@@ -66,8 +66,11 @@ services:
       - RESTORE_SNAPSHOT=${RESTORE_SNAPSHOT:-}
       - RESTORE_RECREATE_DB=N
     depends_on:
-      - cardano-node
-      - postgres
+      # Depend on both services to be healthy before starting.
+      cardano-node:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
```

Comment:
@Kammerlo 
healthcheck is not defined for cardano-node and postgres. Is this a problem?

Reply:
No this isn't a problem. We just need the container to be up and running.

Reply:
In that case, shouldn't we use service_started instead of service_healthy?
I am making such a change locally.

Reply:
I am trying it in a preprod environment right now and it wouldn't start otherwise.

```
Aug 14 13:13:51 ip-10-18-1-87.ap-northeast-1.compute.internal docker-compose[6159]: ERROR: for cardano-db-sync  Service "postgres" is missing a healthcheck configuration
Aug 14 13:13:51 ip-10-18-1-87.ap-northeast-1.compute.internal docker-compose[6159]: Encountered errors while bringing up the project.
```